web-api/src/storage/elastic.rs

```diff
@@ -562,7 +562,7 @@ impl storage::DocumentProperty for Storage {
             ["_source", document_id.as_ref()],
             [(
                 "_source_includes",
-                Some(&*format!("properties.{}", property_id)),
+                Some(&*format!("properties.{property_id}")),
```

Comment:
Finally!
Hated the old way :)

Reply:
this is actually possible since a few versions :) now there is a lint to enforce it. the main caveat is that this currently only works with variable names, if you have a functions call or a a object field reference etc the old interpolation syntax is still required :/

Reply:
Probably because it's still a macro of course, would be quite complex to implement e.g. a function call :/